examples/override.py

```diff
@@ -0,0 +1,18 @@
+#!/usr/bin/python
+
+import json
+import requests
+import sys
+
+sdk_key = sys.argv[1]
+exp_key = sys.argv[2]
+var_key = sys.argv[3]
+
+s = requests.Session()
+s.headers.update({'X-Optimizely-SDK-Key': sdk_key})
```

Comment:
nice - I did not know about this way -  I usually put the header as a part of request.